### ISSUE TITLE
Flag/parameter to make sure that human readable code is used

### ISSUE THREAD

**Issue by emattias** [yarnpkg/yarn#6390]
# TLDR
When I go to definition of a npm package in my local dev environment I want to be taken to the human readable code, not the transpiled code.

# Description
Since alot of packages are transpiled and/or minified with for example babel or typescript I would like in my local dev environment for all packages to point to and use the authored human readable code. Often this is just a matter of instead of pointing the `main` field in `package.json` to `src/index.js` instead of `lib/index.js`.

My thinking is that there is either a parameter like for example: `yarn --human-readable` or maybe an environment variable that controls this.
